### PR TITLE
update spa schedule

### PR DESCRIPTION
**Pull request by jackherizsmith**
Also updated assessment criteria for cross-browser testing.

The project references client side routing in the summary and assessment criteria

Also currently no additional resources are linked

These are from FAC19's SPA resources folder:

[How single page applications work](https://blog.pshrmn.com/how-single-page-applications-work/)
[Single page applications and the server](https://blog.pshrmn.com/single-page-applications-and-the-server/)

### CODE REVIEW

File: curriculum/fullstack-app/schedule.md [hunk 1]
```diff
@@ -1,33 +1,40 @@
-# Single-page app Week Schedule
+# Single Page App Week Schedule
```
Comment (sofer) [inline curriculum/fullstack-app/schedule.md:1]: @jackherizsmith why remove the hyphen 🤷 ?

**Comment by oliverjam** [comment]
This looks good except for the hyphen change 😅 

I'm going to fix the project and resources later today

File: curriculum/fullstack-app/schedule.md (hunk 1 above)
Comment (jackherizsmith) [reply curriculum/fullstack-app/schedule.md:1]: No idea I was just getting confused by all the hyphens. It's called fullstack app in the folder and url, is there a reason we use both in different places?

Comment (oliverjam) [reply curriculum/fullstack-app/schedule.md]: It should probs be changed to single-page app everywhere tbh. Fullstack app is a weird term that doesn't really exist

**Pull request merged**